packages/wallets/src/hooks/usePasswordValidation.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+import zxcvbn from 'zxcvbn';
+
+const usePasswordValidation = (password: string) => {
+    const [errorMessage, setErrorMessage] = useState('');
+    const [score, setScore] = useState(0);
+
+    useEffect(() => {
```

Comment:
both of the useEffects have the same array dependency, you can couple them together and in general I suggest to change this hook into a util function.